.gitignore

```diff
@@ -2,4 +2,3 @@ typings/
 node_modules/
 test/bundle.d.ts
 *.js
-
```

Comment:
editorconfig is configured with `insert_final_newline`. Please honor this or it will create further unneeded diffs in future commits ;)